scripts/go.mod

```diff
@@ -48,14 +48,14 @@ require (
 	github.com/xrash/smetrics v0.0.0-20240521201337-686a1a2994c1 // indirect
 	go.etcd.io/bbolt v1.3.11 // indirect
 	go.uber.org/multierr v1.11.0 // indirect
-	golang.org/x/crypto v0.26.0 // indirect
```

Comment:
will rebase on master after https://github.com/nspcc-dev/neo-go/pull/3748 and remove it

Reply:
Fixed, let's rebase.